### PR TITLE
CTCP-5506: update dependency and connectors

### PR DESCRIPTION
**Pull request by kenneth-rayner**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -111,8 +111,8 @@ microservice {
     # otherwise users in production will be pointed to new phase 5 frontends
     features {
       isGuaranteeBalanceEnabled = true
-      isPhase4Enabled = true
-      isPhase5Enabled = false
+      isPhase4Enabled = false
```
Comment (joecefnhedog) [inline conf/application.conf:114]: are these supposed to have changed

**Review by joecefnhedog — COMMENTED** [review]
lgtm, just the flags

File: conf/application.conf (hunk 1 above)
Comment (kenneth-rayner) [reply conf/application.conf]: @joecefnhedog  reverted

**Comment by jackheslop96** [comment]
Ensure this branch is ran against the journey tests locally first, incase those test only connector changes cause any issues

**Review by joecefnhedog — DISMISSED** [review]

**Review by kevincoatsworth — APPROVED** [review]

**Pull request merged**